Libraries/LibWeb/Crypto/CryptoAlgorithms.cpp

```diff
@@ -3259,7 +3259,7 @@ WebIDL::ExceptionOr<GC::Ref<CryptoKey>> ECDSA::import_key(AlgorithmParams const&
             // 1. Perform any key import steps defined by other applicable specifications, passing format, keyData and obtaining key.
             // TODO: support 'applicable specifications'
 
-            // 2. If an error occured or there are no applicable specifications, throw a DataError.
```

Comment:
That's a spec issue, you might want to upstream it
https://w3c.github.io/webcrypto/#concept-parse-an-asn1-structure

Reply:
while its a typo, from a [search](https://github.com/search?q=org%3Aw3c+occured&type=code) I see the upstream uses this a lot  (150 `occured` and 836 `occurred`)

Reply:
It seem like even [before this PR](https://github.com/LadybirdBrowser/ladybird/blob/24e374d7e1b332c991d732879f7b05dc810d055c/Libraries/LibWeb/Crypto/CryptoAlgorithms.cpp#L3865) there was the correct line 7 times somehow and 2 times with  the same typo as the protocol. so if I revert to keep compatibility with the spec , should I revert the original correct lines too?